### DISCUSSION TITLE
Can you avoid duplicate eslint'ing when a single file is used in multiple configurations?

### DISCUSSION THREAD

**Discussion by Jakobud** [webpack/webpack#12932] [Q&A]
I have a directory structure kinda like this:

```
.
├── js/
│   ├── shared.js
│   ├── A.js
│   └── B.js
└── output/
    ├── siteA/
    └── siteB/
```

I have 2 separate Webpack configurations. One builds a bundle for "Site A" and one builds a bundle for "Site B". 

Site A files consist of `A.js` & `shared.js`. Site B files consist of `B.js` and `shared.js`. 

I always run my webpack build using both configurations at the same time, even when watching. My problem is that when something goes wrong in the `shared.js` like it fails linting or whatever, I see the linting error twice in my terminal: once for Site A config and once for Site B config.

Is it possible to somehow avoid seeing the errors twice? Like somehow be able to suppress all errors (linting errors, compiling errors, module resolution errors, etc) until after both configurations are done and then display them in a way that doesn't show duplicates? Like maybe save the errors to an object using the referenced file path as the object keys so that it doesn't create duplicates and then show all errors at the end when webpack is done.

Any sort of ideas in this area?

**Comment by Jakobud** [comment]
I was actually able to solve this by combining my 2 configurations into a single configuration with some clever output path stuff.

**Reply by alexander-akait** [reply]
How you run `eslint`?

**Reply by Jakobud** [reply]
`plugins.push(new EslintWebpackPlugin());`

**Reply by alexander-akait** [reply]
You need open an issue in `eslint-webpack-plugin`
